api/tests/unit/app_analytics/test_tasks.py

```diff
@@ -278,6 +278,74 @@ def test_populate_feature_evaluation_bucket_15m(freezer):
     assert buckets[7].total_count == 15
 
 
+@pytest.mark.freeze_time("2023-01-19T09:00:00+00:00")
+@pytest.mark.django_db(databases=["analytics"])
+def test_populate_feature_evaluation_bucket__upserts_buckets(freezer) -> None:
```

Comment:
Hmm... I wanted to add a comment here to add typing for the `freezer` argument, but it looks as though it's type is perhaps `Union[StepTickTimeFactory, TickingDateTimeFactory, FrozenDateTimeFactory]` (based on [this](https://github.com/spulec/freezegun/blob/master/freezegun/api.py#L697C24-L697C97) and [this](https://github.com/ktosiek/pytest-freezegun/blob/master/pytest_freezegun.py#L45-L46)) although, since we're not passing either of the `tick` or `auto_tick_seconds` arguments, I think the type is actually `FrozenDateTimeFactory`, right?

Reply:
Didn't realize this version of freezegun exports the type. Thanks! Updated annotations across the whole module.